.github/workflows/build.yml

```diff
@@ -1,3 +1,4 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
```

Comment:
This change is not related to the issue, but I noticed VS Code could not guess the schema correctly without this.